api/environments/models.py

```diff
@@ -155,6 +155,7 @@ def get_from_cache(cls, api_key):
                     cls.objects.select_related(*select_related_args)
                     .filter(Q(api_key=api_key) | Q(api_keys__key=api_key))
                     .distinct()
+                    .defer("description")
```

Comment:
We might want to add a comment as to why we're deferring this?

Reply:
I think it's fine because description being a text field is a perfect field to be deferred?

Reply:
Yeah, you're probably right.